dist/index.html

```diff
@@ -208,9 +208,10 @@ <h3>Basic Crossfit</h3>
 
     <!-- Personal Training Section -->
     <section class="personal-training">
+      <div class="overlay"></div>
```

Comment:
Why?

Reply:
for the darkness of image .. but I removed it